engine.py

```diff
@@ -36,6 +59,20 @@ def init_engine(self):
         # determine if this is a tested version:
         is_certified_version = False
         version_str = Application.version()
+
+        try:
+            # Create a _host_info variable that we can update so later usage of
+            # the `host_info` property can benefit having the updated information.
+            self._host_info = {"name": "Softimage", "version": version_str}
+
+            # Actually log the metric
+            self.log_metric("Launched Software")
+
+        except Exception as e:
+            self.log_error("Unexpected error logging a metric")
```

Comment:
I may have asked this before, but should we include the actual exception message in the logged error? I.e., make use of `e`?

Reply:
I've remove declaration of 'e' since the log exception context already holds the exception.

I did updated because I realized I wasn't logging to the tk-softimage.log but only the application. I was assuming that anything I would see in the app was also in the SG log file.

Now, I'm logging to both, I'm not sure about that as nothing else in that 'engine' is logging to 'tk-softimage.log' log file, everything is logged exclusively to the application own log system.

In any case, here is the output when deliberately causing an exception:

```
2017-10-27 10:38:25,657 [20940 ERROR sgtk.env.project.tk-softimage] Unexpected error logging a metric.
Traceback (most recent call last):
  File "C:\shotgun\workspace\tk-softimage\engine.py", line 69, in init_engine
    self.log_metfric("Launched Software")
AttributeError: 'SoftimageEngine' object has no attribute 'log_metfric'
```